redis-server.py

```diff
@@ -61,122 +63,140 @@ def bulk_string_response(length, response):
-            if "PXAT" in set_options:
-                end_time = int(set_options[set_options.index("PXAT") + 1])
-                time_request = end_time - int(time.time() * 1000)
-            # check first for XX and NX conditions are met for set operation;
-            # then check for other options
-            # (ALSO NEED TO IMPLEMENT FUTHER NESTED IF STATEMENTS FOR TIME OPTIONS)
-
-            # XX only sets if key already exists; returns nil reply if set not performed
-            if "XX" in set_options:
-                # GET command returns pervious value if key exists or nil reply if key doesn't exist
-                if "GET" in set_options:
-                    if message[1] in DATA_DICT:
-                        old_value_len = len(DATA_DICT[message[1]])
-                        old_value = DATA_DICT[message[1]]
-                        old_value_bulk_string = bulk_string_response(
-                            old_value_len, old_value
-                        )
-                        DATA_DICT[message[1]] = message[2]
-                        # check for existing time to live previously set; delete current TTL unless specified to keep.
-                        time_dictionary_check(message[1], set_options, time_request)
-                        return old_value_bulk_string
-                    return NIL_REPLY
-                if message[1] in DATA_DICT:
-                    DATA_DICT[message[1]] = message[2]
-                    time_dictionary_check(message[1], set_options, time_request)
-                    return OKAY
-                return NIL_REPLY
-            # NX only sets if key does not exist; returns nil reply if set not performed
-            if "NX" in set_options:
-                if "GET" in set_options:
-                    # returns current value but does not set new value as key already existed; GET overides nil reply from NX
-                    if message[1] in DATA_DICT:
-                        current_value_len = len(DATA_DICT[message[1]])
-                        current_value = DATA_DICT[message[1]]
-                        current_value_bulk_string = bulk_string_response(
-                            current_value_len, current_value
-                        )
-                        return current_value_bulk_string
-                    # sets new key:value but returns nil as GET was used and no key:value existed previously
-                    if message[1] not in DATA_DICT:
-                        DATA_DICT[message[1]] = message[2]
-                        time_dictionary_check(message[1], set_options, time_request)
-                        return NIL_REPLY
-                if message[1] not in DATA_DICT:
-                    DATA_DICT[message[1]] = message[2]
-                    time_dictionary_check(message[1], set_options, time_request)
-                    return OKAY
-                return NIL_REPLY
+    if command == "SET":
+        command_value = message[2]
```

Comment:
This should go after the `<= 2` check _just in case_ there are only two entries in this list